nselib/data/http-sql-errors.lst

```diff
@@ -39,13 +46,14 @@ MySQL ODBC
 ODBC DB2
 ODBC Driver
 ODBC Error
-ODBC Microsoft Access
+ODBC Microsoft Access Driver
```

Comment:
What is the reference for changing this? This makes the match more specific; is the word "Driver" always included?